bio-qiime-pipeline/src/utils.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional, Tuple
+from pathlib import Path
+
+import csv
+
+from coretex import CustomSample, CustomDataset, Experiment
+from coretex.bioinformatics import ctx_qiime2
+
+
+def summarizeSample(sample: CustomSample, outputDir: Path) -> Path:
+    demuxPath = Path(sample.path) / "demux.qza"
+    visualizationPath = outputDir / "demux.qzv"
+
+    ctx_qiime2.demuxSummarize(str(demuxPath), str(visualizationPath))
+    return visualizationPath
+
+
+def determineTruncLen(sample: CustomSample, forward: bool) -> int:
+    sample.unzip()
+    summariesFileName = "forward-seven-number-summaries.tsv" if forward else "reverse-seven-number-summaries.tsv"
```

Comment:
where do those file names come from?

Reply:
The files are created by the `qiime demux summarize` command and always have the same names

Reply:
can we make constants with these two instead of having them hardcoded here?